core/primitives/src/epoch_manager.rs

```diff
@@ -114,7 +133,7 @@ impl AllEpochConfig {
 #[derive(Debug, Clone, SmartDefault, PartialEq, Eq)]
 pub struct ValidatorSelectionConfig {
     #[cfg(feature = "protocol_feature_chunk_only_producers")]
-    #[default(300)]
+    #[default(200)]
```

Comment:
do we have to hardfork shardnet after this?

Reply:
Good point, fixed